spec/compiler/crystal/tools/init_spec.cr

```diff
@@ -146,7 +150,11 @@ module Crystal
 
         with_file "example_app/shard.yml" do |shard_yml|
           parsed = YAML.parse(shard_yml)
-          parsed["targets"].should eq({"example_app" => {"main" => "src/example_app.cr"}})
+          parsed["targets"].should eq({
+            yaml("example_app") => {
+              yaml("main") => "src/example_app.cr",
+            },
```

Comment:
I think this is not very intuitive. What prevents us from not modifying equality at all in this PR?

Reply:
That is, I'd like this change to be gone but still have it compiling and working.